sktree/datasets/hyppo.py

```diff
@@ -197,31 +197,41 @@ class distribution approaches the first class distribution by a factor of :math:
     )
     # the parameters used for each Gaussian in the mixture for each Marron Wand simulation
     norm_params = MarronWandSims(n_dim=n_informative, cov=cov)(simulation)
+    if seed:
+        rngs = [np.random.default_rng(seed=seed + i) for i in range(n_samples // 2)]
+    else:
+        rngs = [rng for _ in range(n_samples // 2)]
```

Comment:
Do we not need to also re-seed here? or is `None` handle that?

Reply:
I think we only care about this when we fix a seed right?

Reply:
Oh I see. Because if we don't fix the seed then each run will be different anyways?

Reply:
yep